pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1122,6 +1123,10 @@ public static class Dedup {
       public static final String DEFAULT_ALLOW_DEDUP_CONSUMPTION_DURING_COMMIT =
           "default.allow.dedup.consumption.during.commit";
     }
+
+    public static final String QUERY_WORKLOAD_ENABLED = "pinot.server.queryWorkloadEnabled";
```

Comment:
I don't think you need a separate server Config here. We can rely on the configs I'll be creating.